.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
```

Comment:
The script should be `  - mvn clean verify -f alma-server/pom.xml -e`

Reply:
Oh mince ! Tu as raison !

Reply:
what

Reply:
Je corrige ça tout de suite !... après mon café

Reply:
english please